pkg/storage/system_intake.go

```diff
@@ -118,6 +130,12 @@ func (s *Store) UpdateSystemIntake(ctx context.Context, intake *models.SystemInt
 			requester_email_address = :requester_email_address,
 			cost_increase = :cost_increase,
 			cost_increase_amount = :cost_increase_amount,
+			contractor = :contractor,
```

Comment:
I'd also suggest adding a DB test that verifies these fields round-trip to/from the database appropriately.